auth_backends/backends.py

```diff
@@ -2,11 +2,18 @@
 
 For more information visit https://docs.djangoproject.com/en/dev/topics/auth/customizing/.
 """
+import datetime
```

Comment:
We follow the [PEP8 style for imports](https://www.python.org/dev/peps/pep-0008/#imports). Imports should be grouped as follows:

1. standard library imports
2. related third party imports
3. local application/library specific imports

Each group should be alphabetized, and you should put a blank line between each group of imports.